recipes/lttbc/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "lttbc" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lttbc-{{ version }}.tar.gz
+  sha256: ef513c1876a9874daa3f291669ed082db3c181d8caefd47bdea9b55b4e28a383
+
+build:
+  number: 0
+  skip: true   # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy
+    - pip
+    - python
+    - setuptools-scm
+  run:
+    - numpy
```

[Should be](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=pin_compatible#building-against-numpy):
```suggestion
    - {{ pin_compatible('numpy') }}
```

---

roger that